src/helpers/FormContext.js

```diff
@@ -2,10 +2,11 @@ import ControlRecordWrapper from './ControlRecordWrapper';
 
 export default class FormContext {
 
-  constructor(formRecords, patient) {
+  constructor(formRecords, patient, parentRecord) {
```

Comment:
what does the "parentRecord" signify? the immediate parent? When is this scenario applicable?

Reply:
Yes, it is an immediate parent. This parent is used as a reference for newly added sections. Please have a look at [openmrs talk](https://talk.openmrs.org/t/rendering-of-addmore-section-or-obsgroup-is-not-working-as-expected-when-there-are-form-conditions-inside-it/23941/4)

Reply:
it is not clear what is formRecords and what is parentRecord, can we call formRecords as rootRecord

Reply:
Updated the code